src/lang/modifyAst/sweeps.spec.ts

```diff
@@ -352,6 +304,180 @@ extrude001 = extrude(profile001, length = 2)`)
       )
       await runNewAstAndCheckForSweep(result.modifiedAst, rustContextInThisFile)
     })
+
+    it('should add an extrude call to a wall', async () => {
+      const code = `sketch001 = startSketchOn(XY)
+profile001 = startProfile(sketch001, at = [0, 0])
+  |> angledLine(angle = 0deg, length = 0.07, tag = $rectangleSegmentA001)
+  |> angledLine(angle = segAng(rectangleSegmentA001) + 90deg, length = 0.07)
+  |> angledLine(angle = segAng(rectangleSegmentA001), length = -segLen(rectangleSegmentA001))
+  |> line(endAbsolute = [profileStartX(%), profileStartY(%)])
+  |> close()
+extrude001 = extrude(profile001, length = 1)
+plane001 = offsetPlane(XZ, offset = 1)
+sketch002 = startSketchOn(plane001)
+profile002 = circle(sketch002, center = [0, 0], radius = 0.1)`
+      const { ast, artifactGraph, sketches } = await getAstAndSketchSelections(
+        code,
+        instanceInThisFile,
+        kclManagerInThisFile,
+        1
+      )
+      const to = getFacesFromBox(artifactGraph, 1)
+      const result = addExtrude({
+        ast,
+        artifactGraph,
+        sketches,
+        to,
+      })
+      if (err(result)) throw result
+      const newCode = recast(result.modifiedAst, instanceInThisFile)
+      expect(newCode).toContain(
+        `extrude002 = extrude(profile002, to = planeOf(extrude001, face = rectangleSegmentA001))`
+      )
+      const error = await mockExecAstAndReportErrors(
+        result.modifiedAst,
+        rustContextInThisFile
+      )
+      expect(error).not.toBeInstanceOf(Error)
```

Comment:
The rest is just identical except for a few util functions, this line is what is new here from what was deleted in https://github.com/KittyCAD/modeling-app/pull/8511, and would have caught #8729